src/main/java/de/tum/in/www1/artemis/service/connectors/BambooService.java

```diff
@@ -677,6 +680,15 @@ private String getCommitHash(Map<String, Object> buildMap, SubmissionType submis
         return result.getFeedbacks();
     }
 
+    private String processResultErrorMessage(final ProgrammingLanguage programmingLanguage, final String message) {
```

Comment:
As soon, as this method gets bigger, i.e. if we want to have some more complex feedback processing, I would move this to its own service. As of now, this would be overkill.